lib/input-plugin-tests.js

```diff
@@ -73,45 +71,57 @@ module.exports = function pluginTests(test, plugin) {
       t.true(imp.hasOwnProperty('type'), 'Input has a `type` property');
       t.true(typeof imp.type === 'string', 'Input type is a string');
 
+      if (imp.hasOwnProperty('options')) {
+        t.true(Array.isArray(imp.options), 'Options is an object');
```

Comment:
If `Array.isArray(imp.options)` is true, then it is _not_ an object, but an array. Should it be an object or an array